### ISSUE TITLE
Feature request: install and autoupdate CSS from userstyles.org

### ISSUE THREAD

**Issue by vitaly-zdanevich** [darkreader/darkreader#751]
It is obvious that many users of Dark Reader have installed [Stylus extension](https://add0n.com/stylus.html) for the same purpose - would be nice to have one extension instead of two for similar functionality.

![image](https://user-images.githubusercontent.com/3514015/46506689-c183ca00-c83d-11e8-9db3-2f7b252bd63e.png)

**Comment by AndrewDaGuy** [comment]
@vitaly-zdanevich There's something similar to that in dark reader currently available in the new design :]

#### Switching to the new design
1. Open the dark reader panel
2. Click `Dev Tools`
3. Go to the `Dark Reader developer tools` window
4. You should see a button named `Preview new design` on the right of the `Apply` button
5. Click it and it should change to `Switch to old design` 
6. BAM! You're on the new design :]

#### Version 5 Preview Design
1. Open the dark reader panel
2. Click `See all options`
3. You can now configure sites :D

For per site configuration just click `Theme for all websites` and you can create themes/per-site themes B)

**Comment by AndrewDaGuy** [comment]
> > For per site configuration just click `Theme for all websites` and you can create themes/per-site themes B)
> 
> This isn't similar. OP's question/request is to have such logic in Dark Reader to install custom styles from sites and apply them.

So like a big ol library of community-made themes for dark reader

**Comment by Gusted** [comment]
> So like a big ol library of community-made themes for dark reader? Sounds epic to me :]

No, not special made to Dark Reader. If someone likes that idea, we should improve the usage of the static engine. Whereby I'd personally would like to help with, to grow an library of static CSS special made to sites. However, this request is special to the USO https://33kk.github.io/uso-archive/.
